### DISCUSSION TITLE
I installed and set up a remote card in a dashboard, looks great (on PC) until i try to open the dashboard with my iphone

### DISCUSSION THREAD

**Discussion by aughtago** [PRProd/HA-Firemote#545] [Q&A] [answered]
attached is a screen shot of the error displayed on iphone. "Custom Element does not exist...."

![IMG_9442](https://github.com/user-attachments/assets/76ab8736-5a73-485a-a1b9-80b52505f547)

**Comment by PRProd** [comment, marked as answer]
Greetings @aughtago!

This is an easy fix.  The Home Assistant cache on your phone just needs to be cleared.  Take a look at [the instructions listed here](https://github.com/PRProd/HA-Firemote/wiki/Force-a-Refresh#ios-home-assistant-app) and you should be seeing your Firemote on your iPhone!

-Doug

**Reply by aughtago** [reply]
thanks. i think i did it different way, but it appears to be working. after rebooting the Home assistant PI didn't work, i tried just closing the app on my phone and re-opening it, now the remote appears as expected.
